tests/wpt/metadata/html/semantics/scripting-1/the-script-element/module/dynamic-import/dynamic-imports-script-error.html.ini

```diff
@@ -1,32 +1,4 @@
 [dynamic-imports-script-error.html]
-  expected: ERROR
-  [import() must reject with different error objects for each import when there is a bad module specifier]
-    expected: FAIL
-
-  [import() must reject with the same error object for each import when there is a evaluation error]
-    expected: FAIL
-
-  [import() must reject when there is a bad module specifier in a dependency]
-    expected: FAIL
-
   [import() must reject with different error objects for each import when there is a instantiation error]
     expected: FAIL
```

Comment:
This is a bug in current module script implementation because I misunderstood the spec.

Instead of saving the error and always using it, we need to return a different instantiation error if it's imported from different script element or `dynamic import`.

I think refactoring this might make this PR harder to review so I'd like to file a follow-up issue for it and fix it separately!

Reply:
Filed #27261